float64 newRate
---
int64 status
